pkg/apis/core/validation/validation.go

```diff
@@ -5316,3 +5316,10 @@ func IsDecremented(update, old *int32) bool {
 	}
 	return *update < *old
 }
+
+func IsNotEmpty(content string) bool {
```

Comment:
I don't think accepting config values with leading/trailing whitespace is something we want to promote to common use. That requires all consumers to do trimmed checks instead of the more readable `len(...) == 0` checks that you'd expect